webpages/module/obs/feeds.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+/**
+ * Extract OPS feeds.
+ *
+ * PHP version 7.1+
+ *
+ * @category Module
+ * @package  PlanZ
+ * @author   James Shields <james@lostcarpark.com>
+ * @license  Zambia Software Licence
+ * @link     https://github.com/LVerhulst4321/PlanZ
+ */
+
+$title = "Staff - Create OBS Feeds";
+require_once __DIR__ . '/../../db_functions.php';
+require_once __DIR__ . '/../../error_functions.php';
+require_once __DIR__ . '/../../render_functions.php';
+require_once __DIR__ . '/../../StaffCommonCode.php';
+require_once 'obs_functions.php';
```

Comment:
No __DIR__ ?

Reply:
You're right. I didn't have __DIR__ because it's in the same directory. I should have used __DIR__, just without the .. parent directory references.

Reply:
Were you planning on changing that, or just leave it as is?

Reply:
Yes, I'm planning to change this, and a couple of other bits in line with the module changes.

Reply:
👍